src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -65,7 +105,7 @@ const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({
             value={userToAutocompleteOption(users.find((user) => user.userId.toString() === projectLead))}
           />
         </Grid>
-        <Grid item xs={12} md={6} sx={{ mt: 1 }}>
+        <Grid item xs={12} md={3} sx={{ mt: 1 }}>
```

Comment:
these two fields should look like the rest at this screen size 
<img width="920" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/58810131-0c7e-45f2-bbb6-fc562e357175">